core/src/main/java/com/coderanch/blackjack/Hand.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.coderanch.blackjack;
+
+import java.util.*;
+
+/**
+ * A hand in a game of Blackjack.
+ */
+public class Hand {
+
+    private static final int HIGHEST_SCORE = 21;
+    private static final int MAX_SCORE_COUNT = 2^4;
+    private static final int HAND_SPLIT_CARD_MAX = 2;
+
+    private List<Card> cards = new ArrayList<>();
+    private boolean isStopped;
+    private Set<Integer> possibleScores;
+    private int bestScore;
+
+    /**
+     * Adds a card to the hand.
+     *
+     * @param card the card being added to the hand.
+     */
+    public void addCard(Card card) {
+        if(isStopped){
```

Comment:
I would name a hand that has been passed on "fixed" rather than "stopped".

However, I don't think it should be the `Hand`'s responsibility to keep track of whether its fixed. It's the responsibility of the Game of Blackjack.